charts/inbucket/templates/ingress.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: "inbucket"
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ include "inbucket.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "inbucket.chart" . }}
+spec:
+  rules:
+    - host: {{ .Values.host | quote }}
```

Comment:
```suggestion
    - host: {{ required "must specify host" .Values.host | quote }}
```

Reply:
That looks like a good improvement! :+1: 

I think I (almost) accomplished the same by defining `host: ` to be empty in the `values.yaml`. However, this one is more "expressive"; tells better what's expected.